extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetTableLocation.java

```diff
@@ -181,12 +188,22 @@ public List<SortColumn> getSortedColumns() {
     @Override
     @NotNull
     protected ColumnLocation makeColumnLocation(@NotNull final String columnName) {
+        final List<String> nameList;
         final String parquetColumnName = readInstructions.getParquetColumnNameFromColumnNameOrDefault(columnName);
-        final String[] columnPath = parquetColumnNameToPath.get(parquetColumnName);
-        final List<String> nameList =
-                columnPath == null ? Collections.singletonList(parquetColumnName) : Arrays.asList(columnPath);
+        if (resolver == null) {
+            final String[] columnPath = parquetColumnNameToPath.get(parquetColumnName);
+            nameList = columnPath == null ? Collections.singletonList(parquetColumnName) : Arrays.asList(columnPath);
+        } else {
+            final ColumnDescriptor columnDescriptor = resolver.mapping().get(columnName);
+            if (columnDescriptor == null) {
+                nameList = List.of(); // empty, will not resolve
```

Comment:
Let's make sure this actually results in supply `ColumnRegion.Null` implementations when a user tries to access the column.

Reply:
Yes, verified.